tests/testthat/test-cov-struct.R

```diff
@@ -0,0 +1,114 @@
+test_that("cov_struct can be produced from string variable names", {
```

Comment:
please add section dividers `# cov_struct ----` etc in the test files, keeping with the conventions